docs/config.md

```diff
@@ -40,30 +57,29 @@ batch_sizes:
   step: 2
 ```
 
-- `<dict>`: a set of key-value pairs. Here is an example:
+- `<dict>`: a set of key-value pairs
 
 ```yaml
 triton_server_flags:
   log_verbose: True
   exit_timeout_secs: 120
 ```
 
-The above YAML declares the value of batch_sizes to be an array `[2, 4, 6]`.
-
-The following sections describe the configuration options for each of the
-subcommands of the Model Analyzer.
+<br>
 
-## Config options for `profile`
+# Config options for profile
 
-### CLI and YAML Config Options
+## CLI and YAML Config Options
 
-A list of all the configuration options supported by both the CLI and YAML
-config file are shown below. Brackets indicate that a parameter is optional. For
-non-list and non-object parameters the value is set to the specified default.
+A list of all the configuration options supported by **both the CLI and YAML**
+config file are shown below.
 
-The CLI flags corresponding to each of the options below are obtained by
-converting the `snake_case` options to `--kebab-case`. For example,
-`profile_models` in the YAML would be `--profile-models` in the CLI.
+- Brackets indicate that a parameter is optional.
+- For non-list and non-object parameters the value is set to the specified default.
+- The CLI flags corresponding to each of the options below are obtained by
+  converting the `snake_case` options to `--kebab-case`.
+  <br>
+  For example, `profile_models` in the YAML would be `--profile-models` in the CLI.
```

Comment:
It would be nice if the link on line 85 actually worked. In yaml it doesn't work (and it doesn't even fit in width of the page. I have to scroll to the bottom of the yaml, scroll right, and then scroll back up to see it)

Reply:
Moved it to a separate line, but unfortunately there's no way to make a link inside a YAML section